### DISCUSSION TITLE
Welcome to Anno2205-NewFrontiers Discussions!

### DISCUSSION THREAD

**Discussion by Taludas** [Taludas/Anno2205-NewFrontiers#1] [Announcements]
## Welcome Martians!

![Anno2205_2024-05-12_21-58-22-92](https://github.com/Taludas/Anno2205-NewFrontiers/assets/64583643/a6305382-79a2-4c5c-8113-c00c13e772e4)

  This is the Github Discussion for the Anno 2205 New Frontiers Mod. You can post here about:
  * Questions about the mod.
  * Share ideas.

If you want to report bugs, please use the "Issues" page.

**Comment by Ultrivy** [comment]
I hold all corporation shares but i can't take over mars... what i have to do ?

**Reply by Taludas** [reply]
You have to update to the latest version v1.0.2
https://github.com/Taludas/Anno2205-NewFrontiers/releases/latest/download/Anno2205_NewFrontiers.zip
Just overwrite the existing data folder.

**Reply by Ultrivy** [reply]
thank you very much :)

**Reply by Ultrivy** [reply]
It is really a GREAT mod !

**Comment by Yukikazze** [comment]
Thank you for the mod ! But i really suck at making the space station is there a cheat sheet somewhere i can copy please ?

**Comment by SER408** [comment]
Is it fun to feel that all other buildings except houses can be upgraded?

**Comment by frozen170902-glitch** [comment]
Hey can you share how to edit the output production? I wanna modified some file for my self, thx taludas
